sdk/keyvault/keyvault-secrets/review/keyvault-secrets.api.md

```diff
@@ -55,7 +55,7 @@ export interface KeyVaultSecret {
 }
 
 // @public
-export const LATEST_STABLE_KEYVAULT_SERVICE_API_VERSION = "7.1";
+export const LATEST_STABLE_KEYVAULT_SERVICE_API_VERSION = "7.0";
```

Comment:
Is this actually what we should do, though? .NET doesn't refer to it as "latest stable version", but just "latest version". In principal, I agree, but we have validation that is either wrong (needs to exclude previews), or we should point at the latest preview. I will ask the arch board.

Reply:
I opened https://github.com/Azure/azure-sdk/issues/1043 to track a decision.

Reply:
So for 4.1-preview, as we discussed in standup, this should be "7.1-preview". The choice of constant variable name is...unfortunate. Maybe add a new one for latest preview? The most important thing is to make it the default for creating a new client. That will be our working assumption until after we resolve the related conversation.

Reply:
Does it matter if it's not exported? I am not exporting this to the public API surface at the moment.